### PR TITLE
Update the release note with persist V3 changes

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
Update the release note with persist V3 changes

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md [hunk 1]
```diff
@@ -151,6 +160,23 @@ To view bug fixes, see the [GitHub milestone for 2201.5.0 (Swan Lake)](https://g
 #### OpenAPI Tool
 
 #### Persist Tool
+- Added following new arguments to `persist init` commands.
+    - `--datastore` - This is used to indicate the preferred database client. Currently, only 'mysql' is supported..
```
Comment (daneshk) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:164]:
```suggestion
    - `--datastore` - This is used to indicate the preferred database client. Currently, only 'mysql' is supported.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md [hunk 2]
```diff
@@ -151,6 +160,23 @@ To view bug fixes, see the [GitHub milestone for 2201.5.0 (Swan Lake)](https://g
 #### OpenAPI Tool
 
 #### Persist Tool
+- Added following new arguments to `persist init` commands.
+    - `--datastore` - This is used to indicate the preferred database client. Currently, only 'mysql' is supported.
+    - `--module` - This is used to indicate the module in which the files are generated.
+
+    e.g: `bal persist init --datastore mysql --module db`
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:167]:
```suggestion
    E.g., `bal persist init --datastore mysql --module db`
```

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md [hunk 3]
```diff
@@ -151,6 +160,23 @@ To view bug fixes, see the [GitHub milestone for 2201.5.0 (Swan Lake)](https://g
...
+- Changed the `persist init` command to create a `persist` directory in the Ballerina project and generate a new definition file(`model.bal`) in the `persist` directory if the file does not exist.
+- Restricted one persist model definition per Ballerina package.
+- Removed the support for `persist push` command and generate SQL script file in the `persist generate` command.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:170]:
```suggestion
- Removed the support for `persist push` command and generate SQL script file in the `persist generate` command.
```
It is not clear what was removed here. Shall we reword?

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md [hunk 4]
```diff
@@ -151,6 +160,23 @@ To view bug fixes, see the [GitHub milestone for 2201.5.0 (Swan Lake)](https://g
...
+- Changed the `persist generate` command to generate all the Ballerina client, types, db_config files and the SQL script file in the `generated/<module_name>` directory.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:171]:
```suggestion
- Changed the `persist generate` command to generate all the Ballerina client, types, `db_config` files, and the SQL script files in the `generated/<module_name>` directory.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md [hunk 5]
```diff
@@ -151,6 +160,23 @@ To view bug fixes, see the [GitHub milestone for 2201.5.0 (Swan Lake)](https://g
...
+- Renamed the generated files to the following names
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:172]:
```suggestion
- Renamed the generated files to the following.
```

**Pull request merged**